README.md

```diff
@@ -38,6 +38,17 @@ class TestModel extends Model
 }
 ```
 
+You can also define enum as nullable:
+
+```php
+    protected $enums = [
+        'status' => [
```

Comment:
The idea here was to stick to the existing cast and rule configuration and use `:` as separator but keep a string.
https://github.com/spatie/laravel-enum/issues/24#issuecomment-539112657
```php
protected $enums = [
    'status' => TestModelStatus::class.':nullable',
];
```
This implementation also handles two different types as config `string` vs `array`. Which makes it harder for the user and package to handle the config.

Reply:
Sorry but for me having this concatenation is ugly :sweat_smile:. I agree it makes it harder but I don't have another solution to provide the nullable option.

Reply:
This concatenation is common in Laravel and that's why we've chosen it. Primary because it's an optional flag. It's the same like the date/datetime cast which accept the format as colon separated value.